Components/FeatureGoogleAnalytics/functions.php

```diff
@@ -2,21 +2,63 @@
 
 namespace Flynt\Components\FeatureGoogleAnalytics;
 
-require_once __DIR__ . '/GoogleAnalytics.php';
+require_once __DIR__ . '/helpers.php';
 
-use Flynt\Components\FeatureGoogleAnalytics\GoogleAnalytics;
 use Flynt\Utils\Options;
+use Timber\Loader;
+use Flynt\Utils\TwigExtensionFlynt;
+use Timber\Timber;
 
-add_action('init', 'Flynt\Components\FeatureGoogleAnalytics\init');
-
-function init()
-{
+add_filter('Flynt/addComponentData?name=FeatureGoogleAnalytics', function ($data) {
     $googleAnalyticsOptions = Options::getGlobal('GoogleAnalytics');
+
     if ($googleAnalyticsOptions) {
-        new GoogleAnalytics($googleAnalyticsOptions);
+        $data['jsonData'] = json_encode([
+            'gaId' => $googleAnalyticsOptions['gaId'],
+            'anonymizeIp' => $googleAnalyticsOptions['anonymizeIp'],
+            'serverSideTrackingEnabled' => isTrackingEnabled($googleAnalyticsOptions['gaId']),
```

Comment:
How about using "isTrackingEnabled' as a variable name, according to the function name? While looking into that function, I'm wondering if you removed the debug mode?

Reply:
I've changed the variable to be called 'isTrackingEnabled' now. I also removed the debug mode from the function. Was this not necessary?

Reply:
The debug mode should still be there.

Reply:
I added the debug mode back now. It's only function would be to see what is pushed to the data layer, which it now logs instead, if the GA ID is set to 'debug'